docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,78 @@
 # vSphere CSI Driver - Offline Volume Expansion
 
-CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The Vanilla vSphere CSI driver supports offline volume expansion which allows a volume to be extended when it is not attached to a node. Currently the support is available only for block volumes.
+CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The vSphere CSI driver currently extends this support for dynamically/statically created offline block volumes only i.e allows a block volume to be extended when it is not attached to a node. Check the [supported features](../supported_features_matrix.md) section to verify if your environment conforms to all the required versions. Note that offline volume expansion is available from vSphere CSI v2.0 onwards in Vanilla Kubernetes and v2.1 onwards in Tanzu Kubernetes Grid (TKG) service.
 
 ## Feature Gate
 
-Expand CSI Volumes feature is promoted to beta in 1.16 so it is enabled by default. For Kubernetes release before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
+Expand CSI Volumes feature was promoted to beta in kubernetes 1.16, therefore it is enabled by default. For Kubernetes releases before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
 
 ## Sidecar Container
 
-An external-resizer sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing ControllerExpandVolume RPC call against a CSI endpoint and updating PersistentVolume object to reflect new size.
+An external-resizer sidecar container implements the logic of watching the Kubernetes API for Persistent Volume claim edits, issuing the ControllerExpandVolume RPC call against a CSI endpoint and updating the PersistentVolume object to reflect the new size. This container has already been deployed for you as a part of the vsphere-csi-controller pod.
 
-The resize sidecar container will be deployed in the vsphere-csi-controller pod.
+## Requirements
 
-If EXPAND_VOLUME node capability is implemented, NodeExpandVolume will be called by Kubelet on the node if NodeExpansionRequired is set to true in ControllerExpandVolumeResponse.
+If you are using the TKG cluster service and your environment adheres to the required kubernetes and vSphere CSI driver versions mentioned above skip this section and directly proceed to the `Expand PVC Example` section below to use this feature.
```

Comment:
this is not right for TKG. Older TKG OVAs with k8s 1.16.8 will not have resizer sidecar AFAIK. Not sure how to message it.
```
root@4206260419b17c5559f7d5ab02249adc [ ~ ]# kubectl get pods -A | grep csi
vmware-system-csi              vsphere-csi-controller-7c688b88dc-4vckp                                     5/5     Running   0          9h
vmware-system-csi              vsphere-csi-node-7ftc7                                                      0/3     Pending   0          9h
vmware-system-csi              vsphere-csi-node-xh56t                                                      3/3     Running   0          9h
root@4206260419b17c5559f7d5ab02249adc [ ~ ]# kubectl version
Client Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.4-2+a00aae1e6a4a69", GitCommit:"a00aae1e6a4a698595445ec86aab1502a495c1ce", GitTreeState:"clean", BuildDate:"2020-04-22T11:35:29Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.8+vmware.1", GitCommit:"2ffb338875e521fd0436d92f19435f111b2c867e", GitTreeState:"clean", BuildDate:"2020-03-26T02:08:27Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
root@4206260419b17c5559f7d5ab02249adc [ ~ ]#
```

Reply:
The resizer sidecar is part of the deployment YAML we use, so it is not tied to the Kubernetes version but the vSphere CSI version the customer uses. In the introduction I have already mentioned that vSphere CSI v2.1 supports volume expansion for TKGS. 

@RaunakShah Correct me if I am wrong here?